csrc/device_lower/pass/alias_memory.cpp

```diff
@@ -483,7 +483,7 @@ class ScopeMap : private kir::IrVisitor {
   }
 
   void handle(kir::IfThenElse* ite) final {
-    NVF_THROW("lower_alias_memory: no support for IfThenElse at this phase.");
+    kir::IrVisitor::handle(ite);
```

Comment:
Can you add a comment that it is implicitly assumed IfThenElse should not affect the analysis?